Makefile

```diff
@@ -1,48 +1,48 @@
-.PHONY: image develop tests tag-and-push docs version package upload upload-spruce license
+SHELL = /bin/bash
 mkfile_path := $(dir $(abspath $(lastword $(MAKEFILE_LIST))))
 
-PYPI_USERNAME ?= __token__
-
-image:
-	MODULE=pinecone ../scripts/build.sh ./
-
-develop:
-	poetry install -E grpc
-
 test-unit:
-	@echo "Running tests..."
 	poetry run pytest --cov=pinecone --timeout=120 tests/unit
 
 test-grpc-unit:
-	@echo "Running tests..."
 	poetry run pytest --cov=pinecone --timeout=120 tests/unit_grpc
 
 make type-check:
 	poetry run mypy pinecone --exclude pinecone/core
 
-version:
-	poetry version
-
-package:
-	poetry build
-
-upload:
-	poetry publish --verbose --username ${PYPI_USERNAME} --password ${PYPI_PASSWORD}
-
-upload-spruce:
-	# Configure Poetry for publishing to testpypi
-	poetry config repositories.test-pypi https://test.pypi.org/legacy/
-	poetry publish --verbose -r test-pypi --username ${PYPI_USERNAME} --password ${PYPI_PASSWORD}
+gcloud-auth:
+	docker run -ti --name gcloud-config gcr.io/google.com/cloudsdktool/google-cloud-cli gcloud auth login
+
+clean:
+	rm -rf downloads
+	rm -rf gen
+
+download:
+	mkdir downloads
+	docker run --rm \
+		--volumes-from gcloud-config \
+		-v ${mkfile_path}:/workspace \
+		-w /workspace \
+		gcr.io/google.com/cloudsdktool/google-cloud-cli \
+		gcloud storage cp gs://api-codegen/_latest downloads --recursive
+
+regenerate: clean download
+	# Remove problematic query param
+	jq 'walk(if type == "array" then map(if type == "object" then select(.name != "filter") else . end) else . end)' downloads/_latest/openapi/pinecone_api.json > tmp.json
```

Comment:
Query params with type `object` seem to break the python generator. See https://github.com/OpenAPITools/openapi-generator/issues/1587